### ISSUE TITLE
JAWS does not announce selected status for combobox in Chrome

### ISSUE THREAD

**Issue by joshferrell** [FreedomScientific/VFO-standards-support#875]
### Summary

Chrome version 131.0.6778.86 
JAWS 2023.2307.37 

I was using the combobox [example from WAI-ARIA](https://codepen.io/joshferrell/full/mdNNKxP) and selection states are not announced in the listbox. However, this is working as expected in Firefox 123.0.

### Expected result

"Favorite fruit listbox, banana selected 3 of 13"

### Actual result

"Favorite fruit listbox, banana 3 of 13"

### Example

https://codepen.io/joshferrell/full/mdNNKxP

#### JAWS version and build number

JAWS 2023.2307.37 
 
#### Operating System and version

Assistiv Labs on Windows 10

#### Browser and version:

Chrome version 131.0.6778.86

**Comment by JAWS-test** [comment]
Duplicate of https://github.com/FreedomScientific/standards-support/issues/440 and https://github.com/FreedomScientific/standards-support/issues/733

**Comment by joshferrell** [comment]
> Duplicate of #440 and #733

You're right this is a duplicate of those. I'm closing this issue in favor of those.
